_data/sites.json

```diff
@@ -24341,6 +24341,16 @@
         ]
     },
 
+    {
+        "name": "Wilson Parking",
+        "url": "https://www.wilsonparking.com.au",
+        "difficulty": "hard",
```

Comment:
I'll change it to easy, since it seems it can all be done without a human on the other side